src/lib.rs

```diff
@@ -1633,7 +1667,7 @@ impl Build {
     /// This will return a result instead of panicking; see expand() for the complete description.
     pub fn try_expand(&self) -> Result<Vec<u8>, Error> {
         let compiler = self.try_get_compiler()?;
-        let mut cmd = compiler.to_command();
+        let mut cmd = compiler.to_command(None);
```

Comment:
Hmmm I wonder if there's any C/C++ that affects the expand of C/C++ program?

I can't think of any specific to C or specific to C++, AFAIK the only way to affect that is via `-D`.

Maybe changing std version would affect that, I recalled that newer C standard introduces more macros.